app/carbonzipper/metrics.go

```diff
@@ -85,6 +86,13 @@ func NewPrometheusMetrics(config cfg.Zipper) *PrometheusMetrics {
 			},
 			[]string{"code", "handler"},
 		),
+		RenderConsistency: prometheus.NewCounterVec(
+			prometheus.CounterOpts{
+				Name: "render_consistency",
+				Help: "Count of rendered data points, partitioned by the consistency state",
```

Comment:
Please add label descriptions. This will make it clear how to use it.

Reply:
I consider this as resolved.